src/entities/UserRoleItem/const/const.js

```diff
@@ -0,0 +1 @@
+export const URL_USER = "admin-user"
```

Comment:
"admin-user" is repeated 3 times. replace with a single constant from the 'shared/const' directory